arangod/Agency/Constituent.cpp

```diff
@@ -98,57 +97,66 @@ term_t Constituent::term() const {
 }
```

Comment:
GitHub will not let me comment on line 66:  should _term be protected by calling term() instead?  I do not know the threading implications.